ETL/Provision/aws_infrastructure/main.tf

```
@@ -0,0 +1,57 @@
+provider "aws" {
```

```suggestion
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 5.0"
    }
  }
}

provider "aws" {
```
Could add but not needed. Found it here:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs